### PR TITLE
Fix Style/RescueModifier offenses

### PR DESCRIPTION
**Pull request by magni-**
From #752.

### CODE REVIEW

File: lib/paper_trail/version_concern.rb [hunk 1]
```diff
@@ -218,7 +218,7 @@ def changeset
 
     # Returns who put the item into the state stored in this version.
     def paper_trail_originator
-      @paper_trail_originator ||= previous.whodunnit rescue nil
+      @paper_trail_originator ||= previous && previous.whodunnit
```
Comment (jaredbeck) [inline lib/paper_trail/version_concern.rb]: I'd rather use `try` here. PT still depends on activesupport >= 3, so we can't use `try!`.

**Comment by jaredbeck** [comment]
> I'd rather use try here.

This is the only necessary change. The rest looks good!

**Comment by batter** [comment]
:+1:

**Comment by jaredbeck** [comment]
Closed by https://github.com/airblade/paper_trail/pull/762

Thanks Paul!

**Pull request closed without merging**